src/components/Viewer/Player/Player.tsx

```diff
@@ -51,8 +52,14 @@ const Player: React.FC<PlayerProps> = ({
     /**
      * Eject HLS attachment if file extension from
      * the IIIF content resource ID is not .m3u8
+     * and format is not one of many m3u8 formats.
      */
-    if (painting.id.split(".").pop() !== "m3u8") return;
+    if (
+      painting.id.split(".").pop() !== "m3u8" &&
+      painting.format &&
+      !hlsMimeTypes.includes(painting.format)
+    )
+      return;
```

Comment:
My brain is slow today, but I wonder if we still need the `&&` to check for `m3u8` in the `id` at all?

```suggestion
    if (!hlsMimeTypes.includes(painting?.format)) return;
```

Reply:
We can do that, it's just backwards incompatible. Like if someone didn't have that format before it would have worked.

Reply:
Very true. I'm unaware of anyone else using Clover with HLS but I cases could be out there.